reservations/models.py

```diff
@@ -116,6 +116,12 @@ def within_application_round_period(self, app_round: ApplicationRound):
             end__lte=app_round.reservation_period_end,
         )
 
+    def within_period(self, period_start, period_end):
```

Comment:
I guess you could call this method from within_application_round_period to reuse this, but this is quite simple, so not really necessary.

Reply:
changed this => merging